src/irclog2html/irclog2html.py

```diff
@@ -388,6 +389,7 @@ def __init__(self, outfile, colours=None):
         self.outfile = io.TextIOWrapper(outfile, encoding=self.charset,
                                         errors='xmlcharrefreplace',
                                         line_buffering=True)
+        self.outfilename = path.basename(outfilename)
```

Comment:
I would prefer
```suggestion
        self.outfilename = os.path.basename(outfilename)
```
without the `import os.path as path` at the top.